components/longSparkArrow/LongSparkArrow.styles.js

```diff
@@ -0,0 +1,107 @@
+import { P } from '../../style/typography';
+import styled from 'styled-components';
+import { colors } from '../../style/colors';
+import { fontWeights } from '../../style/typography';
+import { min } from '../../lib/responsive';
+
+const UnfilledSparkArrow = styled.img`
+  position: absolute;
+  transform: rotate(90deg);
+  z-index: 1;
+  @media ${min.mobile} {
+    top: -1.3em;
+    right: 12em;
+    height: 30em;
+  }
+  @media ${min.tablet} {
+    top: -1em;
+    right: 8.2em;
+    height: 21em;
+  }
+  @media ${min.desktop} {
+    top: 2.5em;
+    right: 5.7em;
+    height: 14em;
+  }
+`;
+
+const FilledSparkArrow = styled.img`
+  position: absolute;
+  transform: rotate(90deg);
+  z-index: 1;
```

Comment:
idea for less repetitive code: these three lines are the same as **lines 8-10**,  you could pull it out into another styled div:
ex.
`const SparkArrow = styled.img'
position: absolute;
transform: rotate(90deg);
z-index: 1;
'
`
Then in Filled/UnfilledSparkArrow can be: 
`styled(SparkArrow)...`